appveyor.yml

```diff
@@ -61,9 +62,12 @@ clone_depth: 1
 install:
  # Activate the NvDA build system Python virtual environment, creating / updating it if necessary.
  - .\venvUtils\ensureAndActivate.bat
+ - if ERRORLEVEL 1 exit %ERRORLEVEL%
  # Save an exact list of the actual Python packages and their versions that got installed, to aide in reproducing a build
  - py -m pip freeze >installed_python_packages.txt
+ - if ERRORLEVEL 1 exit %ERRORLEVEL%
  - appveyor PushArtifact  installed_python_packages.txt
+ - if ERRORLEVEL 1 exit %ERRORLEVEL%

```

Comment:
Rather than do this here, it could be done in the `on_finish` section. Docs:
>Finalize both successful and failed builds:

It would need to first call `.\venvUtils\ensureAndActivate.bat`

Then, this will be testing the dev workflow more closely. EG calling `scons source` sets up the venv and activates.

Reply:
@michaelDCurran What do you think of this suggestion?